src/receivers/HTTPModuleFunctions.ts

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable import/prefer-default-export */
+import { createHmac } from 'crypto';
+import { parse as qsParse } from 'querystring';
+import rawBody from 'raw-body';
+import tsscmp from 'tsscmp';
+import type { Logger } from '@slack/logger';
+import type { IncomingMessage, ServerResponse } from 'http';
+import { CodedError, ErrorCode } from '../errors';
+import { BufferedIncomingMessage } from './BufferedIncomingMessage';
+
+const verifyErrorPrefix = 'Failed to verify authenticity';
+
+export class HTTPModuleFunctions {
```

Comment:
Just to have namespace to organize these functions, I've added this class and put all the functions as static methods to the class. I know some people may not prefer this style but importing a bunch of functions can be even harder. If you have a different idea to easily manage and use a large set of functions, let me know that!